jetty-documentation/src/main/asciidoc/administration/logging/jetty-server-dump.adoc

```diff
@@ -0,0 +1,189 @@
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+[[jetty-server-dump]]
+=== Jetty Server Dump
+
+The dump feature in Jetty provides a good snapshot of the status of the threadpool, select sets, classloaders, and so forth. 
+To get maximum detail from the dump, you need to `setDetailDump(true)` on any `QueuedThreadPools` you are using. 
+You can do this by a direct call if you are embedding Jetty, or by using properties for standalone Jetty.
+
+[[configuring-dump-feature]]
+==== Configuring the Jetty Server Dump
+
+You can request that Jetty do a dump immediately after starting and just before stopping by calling the appropriate setters on the `Server` instance.
+For embedded usage this can be used by calling the setters directly.
+```java
+server.setDumpAfterStart(true);
+server.setDumpBeforeStop(true);
+```
+
+Standalone Jetty uses two properties to control this behaviour which are referenced in `jetty.xml` to call these setters.
+These properties are `jetty.server.dumpAfterStart` and `jetty.server.dumpBeforeStop`.
+
+These can be temporarily enabled by supplying these properties as command line arguments,
+or they can be enabled via the `server.ini` file (see xref:quick-start-configure[]).
+```
+java -jar $JETTY_HOME/start.jar jetty.server.dumpAfterStart=true jetty.server.dumpBeforeStop=true
+```
+
+[[extra-threadpool-info]]
+==== Extra ThreadPool Information
+
+You can get additional detail from the `QueuedThreadPool` if `setDetailedDump(true)` is called on the thread pool instance.
+For embedded usage this can be used by calling the setters directly.
+```java
+threadPool.setDetailedDump(true);
+```
+
+For standalone jetty you can enable the `threadpool` module and configure the `jetty.threadPool.detailedDump` property.
+See xref:startup-modules[] for information on how to enable a module.
+This same property can also be set via the command line the same as the server dump property.
+
+[[dump-tool-via-jmx]]
+==== Using the Dump Feature via JMX
+
+The `dump` method is on the Server instance and many of its nested components (Handlers, Connectors, and so forth). 
+Dumps may be obtained by calling these methods either in code or via JMX (see xref:using-jmx[]).
+
+The Server MBean has a `dump()` method, which dumps everything, plus a `dumpStdErr()` operation that dumps to StdErr rather than replying to JConsole.
+
+[[examing-jetty-distro-dump]]
+==== Explanation of the Dump Key
+
+- +- bean is a java POJO that is contained by the parent object as a bean added with the addBean method.
```

Comment:
Does this list render OK. Should the key entries be quoted?

Reply:
The list renders fine, I have backquoted the titles to make them clearer.